src/Assignments/ProblemIframe.tsx

```diff
@@ -332,7 +332,7 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
             updateSubmitActive();
         } else {
             if (renderedHTML !== '') {
-                logger.error('this problem has no problemMainForm'); // should NEVER happen when renderedHTML is non-empty
+                logger.error('this problem has no problemMainForm', renderedHTML); // should NEVER happen when renderedHTML is non-empty
```

Comment:
I'm going to be a hypocrite here because I think I have done this too

According to winston logger the variable arguments are "meta", which if you use it with an error object is always fine

However if you do logger.error(string,string) on the backend the second string is omitted (I know we are on the frontend here)

When I set up the logger on the frontend I got it to work where it would behave the same as console.error since we were migrating over, however, we should probably avoid it 🤷‍♂️ 

It would just be 
```
logger.error(`this problem has no problemMainForm ${renderedHTML}`)
```